### PR TITLE
Feature/css classes

### PR DESCRIPTION
**Pull request by rafarubim**
#### What is the purpose of this pull request?
Add missing CSS handles: `forgotPasswordLink`, `dontHaveAccount` and `eyeIcon`

#### How should this be manually tested?

You can see the new CSS classes in
https://rafaprtest4--storecomponents.myvtex.com/

#### Types of changes
- [ ] Bug fix (a non-breaking change which fixes an issue)
- [X] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

#### Clubhouse stories
[ch31060]

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Added
+- `forgotPasswordLink`, `dontHaveAccount`, `eyeIcon` CSS handles
```
Comment (kevinch) [inline CHANGELOG.md]: Are you using prettier? There should be a blank line between the section title and the first list item.

File: react/styles.css [hunk 2]
```diff
@@ -78,6 +82,8 @@
 
 .logoutButton {}
 
+.eyeIcon {}
```
Comment (kevinch) [inline react/styles.css]: Useful empty rules?

File: CHANGELOG.md (hunk 1 above)
Comment (rafarubim) [reply CHANGELOG.md]: I am, but I don't know why it's not working with .md 😕

File: react/styles.css (hunk 2 above)
Comment (rafarubim) [reply react/styles.css]: These are empty classes for customization!

**Pull request merged**